salt/cloud/clouds/digital_ocean.py

```diff
@@ -414,12 +447,23 @@ def __query_node_data(vm_name):
             pass
         finally:
             raise SaltCloudSystemExit(str(exc))
-
-    for network in data['networks']['v4']:
-        if network['type'] == 'public':
-            ip_address = network['ip_address']
+     
+    # add DNS records, set ssh_host, default to first found IP, preferring IPv4 for ssh bootstrap script target
+    addr_families, dns_arec_types = (('v4', 'v6'), ('A', 'AAAA'))
+    arec_map = dict(zip(addr_families, dns_arec_types))
```

Comment:
The lint checker Jenkins is running says this is incompatible python 3 code, but I don't think so.

Python 2.7.5 (default, Jun 24 2015, 00:41:19) 
[GCC 4.8.3 20140911 (Red Hat 4.8.3-9)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> addr_families, dns_arec_types = (('v4', 'v6'), ('A', 'AAAA'))
>>> arec_map = dict(zip(addr_families, dns_arec_types))
>>> arec_map
{'v4': 'A', 'v6': 'AAAA'}